script.js

```diff
@@ -23,23 +25,34 @@ function generateWord(){
 
 }
 
+//generateAllWords() is a function that takes all of our names and creates a link!
 function generateAllWords(){
-    var words = ["ashwin", "pavan", "jenny", "danny", "kevin", "phoenix", "samara", "venus", "erica", "jonathan", "samantha", "ana", "jenna", "guido", "luis", "JT", "newsha", "ashley"];
+    var words = ["ashwin", "jenny", "danny", "kevin", "phoenix", "samara", "venus", "erica", "jonathan", "samantha", "ana", "jenna", "guido", "luis", "JT", "newsha", "ashley"];
     for (let i = 0; i < words.length; i++) {
-        var randomLeft = Math.floor(Math.random() * (80)) + 5; 
-        var randomTop = Math.floor(Math.random() * (80)) + 10; 
+        //generate some random percentages and degrees
+        var randomLeft = Math.floor(Math.random() * (6)) - 3; 
+        var randomTop = Math.floor(Math.random() * (10)) - 5; 
         var randomRotate = Math.floor(Math.random() *10); 
 
+        //create a new element for each name in the list 
         var word = document.createElement('a');
         word.innerText = words[i];
         word.id = 'word_' + words[i];
-        word.href = '';
-        word.target = '_blank';
+        word.href = '../' + words[i] + '/index.html';
```

Comment:
Jenna thinks this should be:
`word.href = '../food-town/' + words[i] + '/index.html';`